scripts/eval/oe-eval.sh

```diff
@@ -18,21 +18,28 @@ set -ex
 # hf-upload is an optional flag to upload the results to huggingface for result tracking.
 # e.g.:
 # ./scripts/eval/oe-eval.sh --model-name olmo_17_7b_turbo_sft  --model-location beaker://01J28FDK3GDNA2C5E9JXBW1TP4 --hf-upload
-# ./scripts/eval/oe-eval.sh --model-name llama-3-tulu-2-dpo-8b --model-location allenai/llama-3-tulu-2-8b --hf-upload
+# ./scripts/eval/oe-eval.sh --model-name llama-3-tulu-2-dpo-8b --model-location allenai/llama-3-tulu-2-dpo-8b --hf-upload
 
 # Tulu eval dev suite is:
-# gsm8k::olmo1
-# drop::llama3
-# minerva_math::llama3
-# codex_humaneval
-# codex_humanevalplus
+# gsm8k::tulu
+# drop::tulu
+# minerva_math::tulu
+# codex_humaneval::tulu
+# codex_humanevalplus::tulu
```

Comment:
Does this change lead to big changes in reported numbers? I worry a bit about the leaderboard and comparing chat to non-chat numbers.

Reply:
I do think it's worth us switching to the chat format, but yeah I'm curious what the delta is

In either case when we switch, I think we should remove the current oe-evals from the leaderboard and create new columns for these. We can just rerun them for a subset of the models (or all if it's not too expensive) to ensure they're comparable

Reply:
Yeah, it does lead to a big change in some evals. Here are the numbers (no chat -> chat) for Llama 3 8B Instruct:

MMLU: 0.6740 -> 0.4069
DROP: 0.5060 -> 0.1517
MATH: 0.1680 -> 0.0145

Let's discuss at the next meeting which evaluations to keep.